Formula/gofabric8.rb

```diff
@@ -0,0 +1,22 @@
+class Gofabric8 < Formula
+  desc "CLI for fabric8 running on Kubernetes or OpenShift"
+  homepage "https://github.com/fabric8io/gofabric8/"
+  url "https://github.com/fabric8io/gofabric8/archive/v0.4.71.tar.gz"
+  sha256 "4eacee14e73056933dab263918cc64e45c57a2c7c7575fd23dada2a6ff3c5fe1"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    files = Dir["*"]
+    mkdir buildpath/"src/github.com/fabric8io/gofabric8" do
+      files.each { |f| mv buildpath/f, "." }
+      system "make", "install", "REV=homebrew"
+    end
+    bin.install "bin/gofabric8"
+  end
+
+  test do
+    system "#{bin}/gofabric8", "version"
```

Comment:
This is an interesting topic which I did give some thought to, before going with this (weak) approach.

> Any chance of a stronger test? 

Just to give some context: this is a CLI app which is entirely dependent upon the presence of other (cask, manually or remotely installed) apps to function, so it's difficult to test in a useful way without a completely controlled environment - including sudo access (think full-on integration testing).  

That said, perhaps this is a possibility to consider? I may not be aware of some brew functionality/assumptions to achieve this!

We'd need to ensure that no existing K8/OpenShift environment was running; install and run K8/OS (e.g. via Cask; with some `sudo` work required for xhyve drivers); ensure that the network config is satisfactory - then we could try to connect. Quite a few variables that could make the process squishier than you might prefer.   

> At least assert on the version otherwise so we know the output isn't unexpected failures instead, etc.

The reason I didn't assert any of the text in 'version' was that I was concerned it might break if they decide to change it around. It's a fairly non-standard layout. I'm perfectly happy to do add it, with that caveat mentioned.

BTW - I realise this is less than ideal. In a previous formula I submitted it was much easier to provide a safely self-contained test.